src/context/WalletProvider/KeepKey/components/Label.tsx

```diff
@@ -1,24 +1,26 @@
 import { Button, Input, ModalBody, ModalHeader } from '@chakra-ui/react'
 import { useToast } from '@chakra-ui/toast'
-import { RecoverDevice, ResetDevice } from '@shapeshiftoss/hdwallet-core'
+import { ResetDevice } from '@shapeshiftoss/hdwallet-core'
 import { useState } from 'react'
 import { useTranslate } from 'react-polyglot'
 import { Text } from 'components/Text'
-import { parseIntToEntropy } from 'context/WalletProvider/KeepKey/helpers'
 import { useWallet } from 'hooks/useWallet/useWallet'
 
+import { useKeepKeyRecover } from '../hooks/useKeepKeyRecover'
+
 export const KeepKeyLabel = () => {
   const [loading, setLoading] = useState(false)
   const {
     setDeviceState,
     state: {
-      deviceState: { disposition, recoverWithPassphrase, recoveryEntropy },
+      deviceState: { disposition },
       wallet,
     },
   } = useWallet()
   const toast = useToast()
   const translate = useTranslate()
   const [label, setLabel] = useState('')
+  const sendKeepKeyRecover = useKeepKeyRecover()
```

Comment:
Missing `await` on async function.

Reply:
Naming nitpick, I would just call this `recoverKeepKey()`, it's not clear what the `send` terminology is about here

Reply:
done